src/main/java/com/ebivariation/contigalias/repo/AssemblyRepository.java

```diff
@@ -38,4 +40,7 @@
     Optional<AssemblyEntity> findAssemblyEntityByGenbank(String genbank);
 
     Optional<AssemblyEntity> findAssemblyEntityByRefseq(String refseq);
+
+    Optional<List<AssemblyEntity>> findAssemblyEntitiesByTaxid(long taxid);
```

Comment:
is it possible to return an empty list and an empty optional? do they mean different things? in these cases people usually just return a list.